src/it/java/com/google/checkstyle/test/chapter5naming/rule526parameternames/ParameterNameTest.java

```diff
@@ -43,36 +49,63 @@ protected String getPath(String fileName) throws IOException {
 
     @BeforeClass
     public static void setConfigurationBuilder() throws CheckstyleException {
-        checkConfig = getCheckConfig("ParameterName");
-        format = checkConfig.getAttribute("format");
+        final List<Configuration> configs = getCheckConfigs("ParameterName");
+
+        Assert.assertEquals(configs.size(), 2);
+
+        privConfig = configs.get(0);
```

Comment:
I did not know the order is not guaranteed, sorry. Then how do I distinguish between the two? One has only the 'excludeScope'  attr, and the other has only 'scope'. Correct me if I am wrong, but there is no 'hasAttribute', and calling 'getAttribute' will result in an exception if the attribute is not present. Of course I could catch it, but it seemed to me that it was not the right way to do it. Would it be acceptable?